src/block/block_ckpt.c

```diff
@@ -671,7 +721,7 @@ __ckpt_process(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_CKPT *ckptbase)
      * Record the checkpoint's allocated blocks. Do so before skipping any processing and before
```

Comment:
This function is now recording allocated and discarded blocks, so this comment needs to be updated.